bitnami/mongodb-sharded/values.yaml

```diff
@@ -741,6 +745,9 @@ mongos:
   ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
   ##
   schedulerName:
+  ## Use StatefulSet instead of Deployment when deploying standalone
```

Comment:
There's no "standalone" mode for MongoDB Sharded. I guess you took this from "MongoDB" chart.

Reply:
Thanks. Fixed